### DISCUSSION TITLE
Version 1.6.5 / 1.7

### DISCUSSION THREAD

**Discussion by renato-mmex** [moneymanagerex/moneymanagerex#6100] [General]
There was a version 1.6.5 planned. To my knowledge it contained more than 70 bugs/change requests. 
Now this version has disappeared and there is a version 1.7 in it, however, these requests of 1.6.5 are not included.
Why ?

**Comment by tactilis** [comment]
> Why ?

The addition of Tags is a major new feature which calls for a larger increment in the version number to 1.7.

Everything else will be included in that release.
